lib/login.dart

```diff
@@ -63,6 +63,8 @@ class _LoginState extends State<Login> {
                                 backgroundColor: Colors.grey,
                                 gravity: ToastGravity.BOTTOM,
                               );
+                              emailController.clear();
```

Comment:
Do not clear the credentials as of now. This functionality will be adapted later.